tests/src/python/test_qgsserver_wms_dxf.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""QGIS Unit tests for QgsServer WMS GetPrint.
+
+From build dir, run: ctest -R PyQgsServerWMSGetMapDxf -V
+
+
+.. note:: This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+"""
+__author__ = 'Tudor Bărăscu'
+__date__ = '27/01/2021'
+__copyright__ = 'Copyright 2020, The QGIS Project'
+
+import os
+
+import urllib.parse
+
+from qgis.testing import unittest
+
+from test_qgsserver import QgsServerTestBase
+
+from qgis.core import (QgsVectorLayer)
+
+# Needed on Qt 5 so that the serialization of XML is consistent among all executions
+os.environ['QT_HASH_SEED'] = '1'
+
+
+class PyQgsServerWMSGetMapDxf(QgsServerTestBase):
+
+    def test_dxf_export_works_with_reverse_axis_epsg(self):
+        qs = "?" + "&".join(["%s=%s" % i for i in list({
```

Comment:
Are you sure about this? I find it hard to follow but it looks like you are assigning the same value to both ends of `=`

Reply:
I'm sure as it's in almost all the server tests (e.g. test_qgsserver_wms_getmap*). Should I do it differently?

Reply:
Nevermind, it's correct because `i` is a tuple.